packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.tsx

```diff
@@ -17,9 +18,10 @@ import {useHistory} from 'react-router-dom'
  * This flag only applies if storefront is ran in Runtime Admin iframe.
  * @param {function(): string | Promise<string>} getToken - A method that returns the access token for the current user
  */
-export const StorefrontPreview = ({enabled = true, getToken}) => {
+export const StorefrontPreview = ({children, enabled = true, getToken}) => {
```

Comment:
Since we're already `export default StorefrontPreview`, then we don't need to export this one too.
```suggestion
const StorefrontPreview = ({children, enabled = true, getToken}) => {
```

Reply:
The names export is needed for the re-export in the index file. That wasn't a mistake.

Reply:
Ah ok, TIL.